ding/model/template/wqmix.py

```diff
@@ -0,0 +1,323 @@
+from typing import Union, List
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from functools import reduce
+from ding.utils import list_split, MODEL_REGISTRY
+from ding.torch_utils.network.nn_module import fc_block, MLP
+from ding.torch_utils.network.transformer import ScaledDotProductAttention
+from .q_learning import DRQN
+
+
+class Mixer(nn.Module):
```

Comment:
If this `Mixer` is the same as that in `qmix.py`, you should import it rather than implement it again